addons/skin.estuary/xml/AddonBrowser.xml

```diff
@@ -20,6 +20,38 @@
 				<include content="ListThumbInfoPanel">
 					<param name="fallback_image" value="DefaultAddon.png" />
 				</include>
+				<control type="group">
+					<visible>!String.IsEqual(ListItem.AddonLifecycleType,normal)</visible>
+					<control type="image">
+						<left>64</left>
+						<top>109</top>
+						<width>472</width>
+						<height>472</height>
+						<texture colordiffuse="AAFFFFFF">colors/black.png</texture>
+					</control>
+					<control type="textbox">
+						<left>85</left>
+						<top>110</top>
+						<width>425</width>
+						<height>470</height>
+						<align>center</align>
+						<aligny>center</aligny>
+						<visible>String.IsEqual(ListItem.AddonLifecycleType,deprecated)</visible>
+						<label>$LOCALIZE[24049]</label>
+						<font>font36_title</font>
+					</control>
+					<control type="textbox">
```

Comment:
instead of duplicating the textbox, you can remove the visible condition and use variable for the label.
`<label>$VAR[AddonLifecycleType]</label>`

and in Variables.xml:
```
<variable name="AddonLifecycleType">
	<value condition="String.IsEqual(ListItem.AddonLifecycleType,$LOCALIZE[xxx])">$LOCALIZE[24049]</value>
	<value condition="String.IsEqual(ListItem.AddonLifecycleType,$LOCALIZE[yyy])">$LOCALIZE[24096]</value>
</variable>
```

Reply:
Yeah, that makes sense :+1: